package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cplace/asc",
-  "version": "0.14.2",
+  "version": "0.15.0",
```

Comment:
Maybe we should be louder here and use version 1?

Reply:
@smasala I agree. This should have been done even before.